src/main/java/io/fabric8/jenkins/openshiftsync/PodTemplateUtils.java

```diff
@@ -5,14 +5,22 @@
 import io.fabric8.kubernetes.api.model.ConfigMap;
 import io.fabric8.kubernetes.api.model.ObjectMeta;
 import io.fabric8.kubernetes.api.model.Pod;
-import io.fabric8.openshift.api.model.*;
+import io.fabric8.openshift.api.model.Image;
+import io.fabric8.openshift.api.model.ImageStream;
+import io.fabric8.openshift.api.model.ImageStreamStatus;
```

Comment:
Can you optimize imports?
it looks like ImageStreamStatus is not used ? is it ?

Reply:
It is used in PodTemplateUtils itself